src/flows/components/header/MenuButton.tsx

```diff
@@ -45,7 +43,7 @@ const MenuButton: FC<Props> = ({handleResetShare}) => {
     event('clone_notebook', {
       context: 'notebook',
     })
-    const clonedId = await clone(flow.id)
+    const clonedId = await cloneNotebook()
```

Comment:
if we're gonna be awaiting, we'd want the loading state for these as well so they know that their request has been sent and they dont keep smashing the submit button. idk if that's a component level thing or a context level thing

Reply:
k. I'd say it's a component layer thing so that the button is responsive to the load